### PR TITLE
Modifying files

### PR DESCRIPTION
**Pull request by Edgar-Degas**

### CODE REVIEW

File: pulsar_spectra/catalogue_papers/Spiewak_2022.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
  "J0023+0923": {
   "Frequency MHz": [
-   1284
```
Comment (NickSwainston) [inline pulsar_spectra/catalogue_papers/Spiewak_2022.yaml:4]: From the text: "All observations included in this paper were performed with
the MeerKAT L-band receiver (856–1 712 MHz) with the Pulsar
Timing User Supplied Equipement (PTUSE) back-end that uses
coherent dedispersion."
From this, the centre frequency is 1284 MHz and the 1400 MHz used on the tables is just an approximation to common values. If you agree with me, please change these values back to 1284.

**Review by NickSwainston — CHANGES_REQUESTED** [review]
Thank you for your eagle-eyed corrections! There is only one change that I disagree with. Once you revert the change or convince me that you are correct, I will approve my review and merge in your branch

**Comment by NickSwainston** [comment]
## Catalogue updates
### Updated publications
 - [Lorimer et al. (1995)](https://ui.adsabs.harvard.edu/abs/1995ApJ...439..933L)
 - [Lorimer et al. (2006)](https://ui.adsabs.harvard.edu/abs/2006MNRAS.372..777L/abstract)
 - [Lorimer et al. (2007)](https://ui.adsabs.harvard.edu/abs/2007MNRAS.379..282L)
 - [Lynch et al. (2012)](https://ui.adsabs.harvard.edu/abs/2012ApJ...745..109L)
 - [Manchester et al. (1993)](https://ui.adsabs.harvard.edu/abs/1993ApJ...403L..29M/abstract)
 - [Mikhailov et al. (2016)](https://ui.adsabs.harvard.edu/abs/2016A%26A...593A..21M/abstract)

File: pulsar_spectra/catalogue_papers/Spiewak_2022.yaml (hunk 1 above)
Comment (Edgar-Degas) [reply pulsar_spectra/catalogue_papers/Spiewak_2022.yaml]: Yes, I changed it back. Thank you for pointing that out.

**Review by NickSwainston — APPROVED** [review]

**Pull request merged**